src/content/docs/ra-primer/ra-in-action.mdx

```diff
@@ -56,7 +56,7 @@ In what follows, we will work out some queries for the following three relations
 In the tables below, we won't repeat the domain of the attributes, ie `String`/`Integer`.
 
 :::note[Remember]
-A **relation** can be viewed as a **table**. The table contains any number of **rows**/**tuples**, and also has an associated **heading** which describes the **attributes** of the tuples. A relation *can not* contain any duplicate tuples.
+A **relation** can be represented as a **table**. The table contains any number of **rows**/**tuples**, and also has an associated **heading** which describes the **attributes** of the tuples. A relation *can not* contain any duplicate tuples. Also, the order in which rows and columns are shown has no particular meaning.
```

Comment:
For rows, no discussion. For columns... Codd treated position as important. How about The Third Manifesto? Do they explicitly define attributes as unordered? (In Bmg they are of course.)

Reply:
TTM very explicitly prohibits columns ordering. In addition to conveying a very implicit semantics (!), the whole algebra becomes a (relative) nightmare, notably because it prevents join's commutativity, and/hence lots of mathematics-driven join optimizations.